### ISSUE TITLE
options have no effect on AI behavior

### ISSUE THREAD

**Issue by medovina** [minception/Evora#18]
I've found that specifying options seems to have no effect on the behavior of an AI such as MinimaxAI.

To investigate, I added this debugging statement at the beginning of the for loop in ai::minimax_ai::init():

```
   for (auto && arg : args)
	{
        printf("arg.first = '%s'\n", arg.first.c_str());   // DEBUG
        if(arg.first == "time")
        ...
```

Now, when I run

```
$ build/ConsoleUI/azul_console MinimaxAI depth=1 GreedyAI 10
```

it prints

```
arg.first = '=1'
```

That doesn't seem right.

**Comment by medovina** [comment]
The bug is in parse_args() in ConsoleUI.cpp.  To fix it, you can change

```
ai1_args.emplace_back(arg_list[index].substr(delim_pos), arg_list[index].substr(delim_pos + 1, arg_list[index].size()));
```

to

```
ai1_args.emplace_back(arg_list[index].substr(0, delim_pos), arg_list[index].substr(delim_pos + 1, arg_list[index].size()));
```

(and similarly for ai2, obviously).

**Comment by minception** [comment]
fixed in 03d4acc
